pkg/datapath/linux/routing/routing.go

```diff
@@ -134,28 +134,37 @@ func (info *RoutingInfo) InstallRoutes(mtu int, compat bool) error {
 }
 
 func (info *RoutingInfo) installRoutes(ifindex, tableID int) error {
-
 	// Nexthop route to the VPC or subnet gateway
```

Comment:
Would it make sense to have a variant of `InstallRoutes` that use `Table[Route]` to check if we need to do the syscall?